### PR TITLE
Add option to disable writing a meas_date event

### LINKED ISSUE #31: Exported file contains additional event
When exporting a file with events, the resulting .vmrk file has one additional event with the measurement date. This is not ideal because when I read the file back in, I suddenly have one additional event as compared to the original data. Isn't there another way to store the measurement date? If not, could we add an option to disable storing the additional event (e.g. use `meas_date=None` for this purpose instead of writing a zero date, or define a new value such as `meas_date="none"`)?

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #31.

### CODE REVIEW

**Comment by cbrnr** [comment]
This is ready for merge (if you are OK with changing the meaning of `meas_date=None`).

**Comment by sappelhoff** [comment]
Looks good to me, also that the meaning of "None" is changed.

Can you please document this in https://github.com/bids-standard/pybv/blob/master/docs/changelog.rst ?

After that, +1 to merge

**Comment by cbrnr** [comment]
Alright, done (not sure if I got the authors list correct because I assumed you list only the authors contributing to a specific release - if all authors should be listed LMK).

**Comment by sappelhoff** [comment]
We are adding the "authors" section only with the release commit :-) so you can leave that empty for now and prior to the next release, we'll update it

**Comment by cbrnr** [comment]
Understood - ready!

File: docs/changelog.rst [hunk 1]
```diff
@@ -15,6 +15,14 @@ Here we list a changelog of pybv.
 Current
 =======
 
+Changelog
+~~~~~~~~~
+- Add option to disable writing a meas_date event (which is also the new default) by `Clemens Brunner`_ (`#32 <https://github.com/bids-standard/pybv/pull/32>`_)
+
+Authors
+~~~~~~~
```
Comment (sappelhoff) [inline docs/changelog.rst]:
```suggestion

```

File: docs/changelog.rst [hunk 2]
```diff
@@ -15,6 +15,14 @@ Here we list a changelog of pybv.
 Current
 =======
 
+Changelog
+~~~~~~~~~
+- Add option to disable writing a meas_date event (which is also the new default) by `Clemens Brunner`_ (`#32 <https://github.com/bids-standard/pybv/pull/32>`_)
+
+Authors
```
Comment (sappelhoff) [inline docs/changelog.rst]:
```suggestion

```

**Comment by sappelhoff** [comment]
thanks @cbrnr !

**Pull request merged**